certified-connectors/Evocom/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2416 @@
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "description": "Task type id",
+            "x-ms-summary": "Type"
+          }
+        ]
+      }
+    },
+    "/api/Schemas/ChangeTypes": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Get taskchange-types",
+        "description": "Gets a list of available taskchange-types",
+        "operationId": "GetChangeTypes",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "type",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "description": "Change type id",
+            "x-ms-summary": "Type"
+          }
+        ]
+      }
+    },
+    "/api/Schemas/TaskStatus": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Get task status",
+        "description": "Gets a list of available task-status",
+        "operationId": "GetStatus",
+        "x-ms-visibility": "internal",
+        "parameters": []
+      }
+    },
+    "/api/Route/Definition/Steps/{id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StepsResponse"
+            }
+          }
+        },
+        "summary": "Get template steps",
+        "description": "Get process template steps",
+        "operationId": "GetTemplateSteps",
+        "parameters": [
+          {
+            "name": "x-ep-tenant",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Select a tenant",
+            "x-ms-summary": "Tenant",
+            "x-ms-dynamic-values": {
+              "operationId": "GetTenants",
+              "value-path": "tenantId",
+              "value-title": "name"
+            },
+            "x-ms-dynamic-list": {
+              "operationId": "GetTenants",
+              "itemValuePath": "tenantId",
+              "itemTitlePath": "name"
+            }
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Enter a template id",
+            "x-ms-summary": "Template id"
+          }
+        ]
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://evocom.de"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://evocom.de/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Collaborative;Processes;Projects;Tasks;MostEasy;UX"
```

Comment:
We have a maximum of two categories allowed per connector, and they must be chosen from the following list of options:

`AI, Business Management, Business Intelligence, Collaboration, Commerce, Communication, Content and Files, Finance, Data, Human Resources, Internet of Things, IT Operations, Lifestyle and Entertainment, Marketing, Productivity, Sales and CRM, Security, Social Media, Website.`